deploy/complete/kubernetes/mushop.yaml

```diff
@@ -1239,7 +1240,7 @@ spec:
             - name: ZIPKIN
               value: zipkin.jaeger.svc.cluster.local
             - name: JAVA_OPTS
-              value: -Xms32m -Xmx150m -XX:MaxRAM=150m -Djava.security.egd=file:/dev/urandom -Doracle.jdbc.fanEnabled=false -XX:+UnlockExperimentalVMOptions -XX:+UseZGC -Dlogging.level.mushop.orders=INFO -Dspring.zipkin.enabled=false
+              value: -Xms64m -Xmx300m -XX:MaxRAM=300m -Djava.security.egd=file:/dev/urandom -Doracle.jdbc.fanEnabled=false -XX:+UnlockExperimentalVMOptions -XX:+UseZGC -Dlogging.level.mushop.orders=INFO -Dspring.zipkin.enabled=false
```

Comment:
The memory usage is already high, and is needed to low to fit lower shapes and preparation for Always-free support. If need to increase for a specific use case, need to include as variable and change on demand, keeping the low entry as default.

Reply:
I had faced java heal space issue and had to increase. I am considering to implement new service.

Reply:
150m is actually too high. Orders is the bigger offender, catalogue for example uses 17Mb in load. MuShop runs on micro shapes and externally on small infra (1.1, Raspberry PI, arm, ...)